cpp/include/llio.h

```diff
@@ -126,25 +128,22 @@ class message {
   size_t _size;
 };
 
+#define _SIZE0 32
```

Comment:
a `static constexpr size_t size0 = 32` in `message_builder` (private?) is preferrable over `#define`'s